### PR TITLE
daniel first attempt.

### PR DESCRIPTION
**Pull request by danielfa3141**
notice i removed the "DropOut" which supposed to improve convergence (I read briefly)

### CODE REVIEW

File: transfer/dummy.py [hunk 1]
```diff
@@ -23,24 +30,39 @@ def main():
...
 
+    model.add(tf.keras.models.Conv2D(32, kernel_size=(3, 3), activation='relu',
+                     input_shape=input_shape))
+
+    model.add(tf.keras.models.Conv2D(64, (3, 3), activation='relu'))
+
+    # reduce dim, choose best
+    model.add(tf.keras.models.MaxPooling2D(pool_size=(2, 2)))
+
+    #model.add(Dropout(0.25))
```
Comment (guy120494) [inline transfer/dummy.py]: remove

File: transfer/dummy.py [hunk 2]
```diff
@@ -23,24 +30,39 @@ def main():
...
+    model.add(tf.keras.models.Flatten())
+
+    model.add(tf.keras.models.Dense(128, activation='relu'))
+
+    #model.add(Dropout(0.5))
```
Comment (guy120494) [inline transfer/dummy.py]: remove

File: transfer/dummy.py [hunk 3]
```diff
@@ -23,24 +30,39 @@ def main():
...
+    model.add(tf.keras.models.Dense(NUMBER_OF_LABELS, activation='softmax'))
+    model.compile(loss=tf.keras.losses.categorical_crossentropy, optimizer=tf.keras.optimizers.SGD, metrics=['accuracy'])
+
+    history = model.fit(x=train_data, y=train_labels, batch_size=10, epochs=16, verbose=1)
```
Comment (guy120494) [inline transfer/dummy.py]: is batch_size 10 ideal?

File: transfer/dummy.py [hunk 4]
```diff
@@ -23,24 +30,39 @@ def main():
...
     result = model.predict(test_data[0:10])
```
Comment (guy120494) [inline transfer/dummy.py]: you can delete the result print, we draw a graph instead

**Comment by guy120494** [comment]
How many layers do you think we need? in order to analyze transfer learning, we can't have only a few layers.
maybe 7 or 8 layers will do?

File: transfer/dummy.py (hunk 4 above)
Comment (danielfa3141) [reply transfer/dummy.py]: removed printing

**Comment by danielfa3141** [comment]
i think 8 layers top

File: transfer/dummy.py [hunk 5]
```diff
@@ -25,26 +26,30 @@ def main():
...
+    model.add(tf.keras.models.Flatten())
+
+    model.add(tf.keras.models.Dense(128, activation='relu'))
+
+    #model.add(Dropout(0.5))
+    model.add(tf.keras.models.Dense(NUMBER_OF_LABELS, activation='softmax'))
+    model.compile(loss=tf.keras.losses.categorical_crossentropy, optimizer=tf.keras.optimizers.SGD, metrics=['accuracy'])
+
+    history = model.fit(x=train_data, y=train_labels, batch_size=128, epochs=10, verbose=1,
+                        validation_data=(test_data, test_labels))
```
Comment (guy120494) [inline transfer/dummy.py]: the test_data should not be here. you can use validation_split = 0.2 or 0.1 instead
